theano/gradient.py

```diff
@@ -6,7 +6,7 @@
 import time
 import warnings
 
-import numpy  # for numeric_grad
+import numpy as np # for numeric_grad
```

Comment:
>"you need to change numpy to np in elemwise.py line 490"

Thank you for your patience. I have changed numpy to np in elemwise.py in lines 490 and 507. 

Do I need to check all other remaining files?